### PR TITLE
Android SDK/NDK: Move to root folder(C:\) on Windows.

### PR DESCRIPTION
**Pull request by Darleev**
# Description
Bug fixing
Some Android NDK functionality does not work due to spaces in the PATH. In order to fix it, the path has been changed to another value in scope of this PR:

#### Related issue:
https://github.com/actions/virtual-environments/issues/1074

## Check list
- [+] Related issue / work item is attached
- [-] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/win/scripts/Installers/Update-AndroidSDK.ps1 [hunk 1]
```diff
@@ -34,7 +34,7 @@ Expand-Archive -Path .\android-sdk-licenses.zip -DestinationPath 'C:\Program Fil
 # run the updates.
 # keep newer versions in descending order
 
-$sdk_root = "C:\Program Files (x86)\Android\android-sdk"
+$sdk_root = "C:\Progra~2\Android\android-sdk"
```
Comment (al-cheb) [inline images/win/scripts/Installers/Update-AndroidSDK.ps1:37]: You can break some programs what don't use 8.3 short path. Can we use a separate folder at the root of C:\ without space?

Comment (Darleev) [reply images/win/scripts/Installers/Update-AndroidSDK.ps1]: I agree with you. Thank you. PR has been re-worked.

File: images/win/scripts/Installers/Update-AndroidSDK.ps1 [hunk 2]
```diff
@@ -100,7 +101,9 @@ Push-Location -Path $sdk.FullName
     "ndk-bundle"
 
     # Android NDK root path.
-    $ndk_root = "C:\Program Files (x86)\Android\android-sdk\ndk-bundle"
+    $ndk_root = "$sdk_root\ndk-bundle"
+    # Create symlink for backward compatibility
+    mklink /J "C:\Program Files (x86)\Android\android-sdk" $sdk_root
```
Comment (vmapetr) [inline images/win/scripts/Installers/Update-AndroidSDK.ps1:106]: Could we use environment variable `$env:ProgramFiles(x86)`, instead of hardcoded path?

Comment (Darleev) [reply images/win/scripts/Installers/Update-AndroidSDK.ps1]: Thank you, added.

File: images/win/scripts/Installers/Update-AndroidSDK.ps1 [hunk 3]
```diff
@@ -100,7 +101,9 @@ Push-Location -Path $sdk.FullName
     "ndk-bundle"
 
     # Android NDK root path.
-    $ndk_root = "C:\Program Files (x86)\Android\android-sdk\ndk-bundle"
+    $ndk_root = "$sdk_root\ndk-bundle"
+    # Create symlink for backward compatibility
+    mklink /J "$env:ProgramFiles(x86)\Android\android-sdk" $sdk_root
```
Comment (miketimofeev) [inline images/win/scripts/Installers/Update-AndroidSDK.ps1:106]: ${env:ProgramFiles(x86)}

File: images/win/scripts/Installers/Update-AndroidSDK.ps1 [hunk 4]
```diff
@@ -100,7 +101,9 @@ Push-Location -Path $sdk.FullName
     "ndk-bundle"
 
     # Android NDK root path.
-    $ndk_root = "C:\Program Files (x86)\Android\android-sdk\ndk-bundle"
+    $ndk_root = "$sdk_root\ndk-bundle"
```
Comment (miketimofeev) [inline images/win/scripts/Installers/Update-AndroidSDK.ps1:104]: `Join-Path $sdk_root "ndk-bundle"`

File: images/win/scripts/Installers/Update-AndroidSDK.ps1 (hunk 3 above)
Comment (Darleev) [reply images/win/scripts/Installers/Update-AndroidSDK.ps1]: Thanks

File: images/win/scripts/Installers/Update-AndroidSDK.ps1 (hunk 4 above)
Comment (Darleev) [reply images/win/scripts/Installers/Update-AndroidSDK.ps1]: Fixed, thank you.

File: images/win/scripts/Installers/Update-AndroidSDK.ps1 [hunk 5]
```diff
@@ -11,6 +11,7 @@ $sdkArchPath = Start-DownloadWithRetry -Url "https://dl.google.com/android/repos
 Expand-Archive -Path $sdkArchPath -DestinationPath android-sdk -Force
 
 $sdk = Get-Item -Path .\android-sdk
```
Comment (al-cheb) [inline images/win/scripts/Installers/Update-AndroidSDK.ps1:13]: variable assigned but never used

File: images/win/scripts/Installers/Update-AndroidSDK.ps1 [hunk 6]
```diff
@@ -11,6 +11,7 @@ $sdkArchPath = Start-DownloadWithRetry -Url "https://dl.google.com/android/repos
 Expand-Archive -Path $sdkArchPath -DestinationPath android-sdk -Force
```
Comment (al-cheb) [inline images/win/scripts/Installers/Update-AndroidSDK.ps1:11]: Can we replace to `Extract-7Zip`?

File: images/win/scripts/Installers/Update-AndroidSDK.ps1 [hunk 7]
```diff
@@ -28,13 +29,13 @@ $sdk = Get-Item -Path .\android-sdk
...
 $content = [System.Convert]::FromBase64String($base64Content)
 Set-Content -Path .\android-sdk-licenses.zip -Value $content -Encoding Byte
-Expand-Archive -Path .\android-sdk-licenses.zip -DestinationPath 'C:\Program Files (x86)\Android\android-sdk' -Force
+Expand-Archive -Path .\android-sdk-licenses.zip -DestinationPath $sdk_root -Force
```
Comment (al-cheb) [inline images/win/scripts/Installers/Update-AndroidSDK.ps1:32]: Can we replace to Extract-7Zip?

**Pull request closed without merging**